src/burnchains/db.rs

```diff
@@ -127,10 +207,45 @@ CREATE TABLE burnchain_db_block_ops (
     block_hash TEXT NOT NULL,
     op TEXT NOT NULL,
     txid TEXT NOT NULL,
+
     FOREIGN KEY(block_hash) REFERENCES burnchain_db_block_headers(block_hash)
 );
 
-CREATE TABLE db_config(version TEXT NOT NULL);";
```

Comment:
Why remove schema versioning? We don't need to necessarily use it, but having a schema version will make life easier if the `stacks-blockchain` ever wants to implement schema changes without requiring chainstate resets.

Reply:
I think that was a merge artifact -- definitely wasn't intentional. Will revert.